CMakeLists.txt

```diff
@@ -121,7 +120,6 @@ message("-- ${BoldBlue}rocAL Build Type -- ${CMAKE_BUILD_TYPE}${ColourReset}")
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 list(APPEND CMAKE_PREFIX_PATH ${ROCM_PATH} ${ROCM_PATH}/hip)
 find_package(HALF QUIET)
-find_package(HIP QUIET)
 
```

Comment:
@kiritigowda @SundarRajan28  Removed this line since it's not need at top level and causes the HIP issue noticed by Sundar

Reply:
This should work. I will close PR#156